packages/perseus/src/widgets/categorizer/categorizer.tsx

```diff
@@ -18,12 +18,10 @@ import {getPromptJSON as _getPromptJSON} from "../../widget-ai-utils/categorizer
 import type {Widget, WidgetExports, WidgetProps} from "../../types";
 import type {CategorizerPromptJSON} from "../../widget-ai-utils/categorizer/categorizer-ai-utils";
 import type {PerseusCategorizerWidgetOptions} from "@khanacademy/perseus-core";
-import type {
-    PerseusCategorizerRubric,
-    PerseusCategorizerUserInput,
-} from "@khanacademy/perseus-score";
+import type {CategorizerPublicWidgetOptions} from "@khanacademy/perseus-core/src/widgets/categorizer/categorizer-util";
+import type {PerseusCategorizerUserInput} from "@khanacademy/perseus-score";
 
-type Props = WidgetProps<RenderProps, PerseusCategorizerRubric> & {
+type Props = WidgetProps<RenderProps> & {
```

Comment:
Can `PerseusCategorizerRubric` go away because `props.reviewModeRubric` isn't used in the categorizer widget?

Reply:
After removing `PerseusCategorizerRubric` from Props, there are no other references to `rubric` anywhere in the file.